exercises/run-length-encoding/README.md

```diff
@@ -0,0 +1,59 @@
+# Run Length Encoding
+
+Implement run-length encoding and decoding.
+
+Run-length encoding (RLE) is a simple form of data compression, where runs
+(consecutive data elements) are replaced by just one data value and count.
+
+For example we can represent the original 53 characters with only 13.
```

Comment:
This might be better worded as "In the following example, we can represent..." since the example follows the text.